cvmfs/download.cc

```diff
@@ -177,25 +177,27 @@ static size_t CallbackCurlHeader(void *ptr, size_t size, size_t nmemb,
   //          header_line.c_str());
 
   // Check http status codes
-  if (HasPrefix(header_line, "HTTP/1.", false)) {
-    if (header_line.length() < 10)
+  if (HasPrefix(header_line, "HTTP/", false)) {
+    if (header_line.length() < 12) {
```

Comment:
Oh, you are right... what we should check is if the line is 10 bytes long for HTTP/1.1 and 8 (not 12!) for HTTP/2. Instead of subtracting I added by mistake.
In that case I think both could check a length of 10, in case after some time a "HTTP/2.1", for instance, appears.